app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheet.kt

```diff
@@ -347,23 +347,23 @@ class BloodPressureEntrySheet : BottomSheetActivity(), BloodPressureEntryUi, Rem
 
```

Comment:
Could you reword the commit message to follow the convention please?

Ex: "Add common view flipper animations for blood sugar and blood pressure entry".

Reply:
Done